contracts/RMRK/multiasset/IRMRKMultiAsset.sol

```diff
@@ -19,12 +19,12 @@ interface IRMRKMultiAsset is IERC165 {
     /**
      * @notice Used to notify listeners that an asset object at `assetId` is added to token's pending asset
      *  array.
-     * @param tokenId ID of the token that received a new pending asset
+     * @param tokenIds An array of token IDs that received a new pending asset
      * @param assetId ID of the asset that has been added to the token's pending assets array
      * @param replacesId ID of the asset that would be replaced
      */
     event AssetAddedToToken(
```

Comment:
I'd also rename the event to `AssetAddedToTokens` in order to reflect that this can now impact multiple tokens at once 🤔

Reply:
Agree, it should be plural because as it is now is a bit misleading.

Reply:
I agree with your agreement. :smile: 
Change is done.